lib/omnibus/fetcher.rb

```diff
@@ -44,6 +44,12 @@ class Fetcher
     attr_reader :resolved_version
 
     #
+    # The upstream version as described before resolution.  This
+    # will usually be the same as +resolved_version+ but may refer to
+    # a remote ref name or tag for a source such as git.
+    attr_reader :described_version
```

Comment:
Need an `@return` here for the content type. The general style we use is also to end with an extra comment newline before the attr_reader. See above for examples please.